[AArch64] Mandrel fails to build the release/25.0 branch on GHA

## Description
In the new Aarch64 CI builders we see a build failure on that looks like this:

```
Exception in thread "main" java.lang.RuntimeException: java.lang.RuntimeException: Failed, exit code: 255
	at OperatingSystem.exec(build.java:1884)
	at build.getMandrelVersion(build.java:354)
	at build.main(build.java:76)
Caused by: java.lang.RuntimeException: Failed, exit code: 255
	at OperatingSystem.exec(build.java:1873)
	at build.getMandrelVersion(build.java:354)
	at build.main(build.java:76)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at jdk.compiler/com.sun.tools.javac.launcher.SourceLauncher.execute(SourceLauncher.java:258)
	at jdk.compiler/com.sun.tools.javac.launcher.SourceLauncher.run(SourceLauncher.java:138)
	at jdk.compiler/com.sun.tools.javac.launcher.SourceLauncher.main(SourceLauncher.java:76)
```

See: https://github.com/graalvm/mandrel/actions/runs/17382337324/job/49342432391#step:8:50

Building with `--verbose` I get:

```
DEBUG [OperatingSystem] Execute [/home/runner/work/mandrel/mandrel/mx/mx, version] in /home/runner/work/mandrel/mandrel/mx with environment variables [MX_PYTHON=python]
DEBUG [build] Cleaning mxbuild dirs...
DEBUG [build] Building...
DEBUG [OperatingSystem] Execute [/home/runner/work/mandrel/mandrel/mx/mx, graalvm-version] in /home/runner/work/mandrel/mandrel/mandrel/substratevm with environment variables [MX_PYTHON=python]
DEBUG [OperatingSystem] Graal requires JDK 26 or later.
DEBUG [OperatingSystem] Set the JVMCI_VERSION_CHECK environment variable to "ignore" to suppress this error or to "warn" to emit a warning and continue execution.
DEBUG [OperatingSystem] Currently used Java home directory is /home/runner/work/mandrel/mandrel/openjdk.
DEBUG [OperatingSystem] Currently used VM configuration is: OpenJDK 64-Bit Server VM
DEBUG [OperatingSystem] Download JDK 26 or later.
DEBUG [OperatingSystem] Command '/home/runner/work/mandrel/mandrel/openjdk/bin/java -Xlog:disable /home/runner/work/mandrel/mandrel/mandrel/compiler/src/jdk.graal.compiler/src/jdk/graal/compiler/hotspot/JVMCIVersionCheck.java' returned non-zero exit status 255.
Exception in thread "main" java.lang.RuntimeException: java.lang.RuntimeException: Failed, exit code: 255
	at OperatingSystem.exec(build.java:1884)
	at build.getMandrelVersion(build.java:354)
	at build.main(build.java:76)
Caused by: java.lang.RuntimeException: Failed, exit code: 255
	at OperatingSystem.exec(build.java:1873)
	at build.getMandrelVersion(build.java:354)
	at build.main(build.java:76)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at jdk.compiler/com.sun.tools.javac.launcher.SourceLauncher.execute(SourceLauncher.java:258)
	at jdk.compiler/com.sun.tools.javac.launcher.SourceLauncher.run(SourceLauncher.java:138)
	at jdk.compiler/com.sun.tools.javac.launcher.SourceLauncher.main(SourceLauncher.java:76)
```

So the problem seems to be that we are trying to build `graal/master` with OpenJDK 25. Fixing it ....

--- a/.github/workflows/weekly.yml
+++ b/.github/workflows/weekly.yml
@@ -71,7 +71,8 @@ jobs:
     uses: ./.github/workflows/base.yml
     with:
       quarkus-version: "main"
-      mandrel-version: "graal/master"
+      mandrel-version: "mandrel/25.0"
+      mandrel-packaging-version: "25.0"
       jdk: "25/ea"
       issue-number: "830"
       issue-repo: "graalvm/mandrel"
@@ -84,7 +85,7 @@ jobs:
   ####
   # Test Quarkus main with Mandrel latest on AArch64
   ####
-  q-main-mandrel-26-latest:
+  q-main-mandrel-26-latest-arm:
     name: "Q main M 26 latest on AArch64"
     uses: ./.github/workflows/base.yml
     with:

